content/en/docs/azure/machinelearningcomponent.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "Azure Machine Learning Components"
+description = "Azure Machine Learning Pipeline Component that performs on step in ML workflow"
+weight = 6
++++
+
+A pipeline component is a self-contained set of code that performs one step in the ML workflow. A component is analogous to a function, in that it has a name, parameters, return value and a body. Azure Machine Learning Components are the pipeline components that would integrate with [Azure Machine Learning](https://docs.microsoft.com/en-us/azure/machine-learning/) to manage the life cycle of your models to improve the quality and consistency of your machine learning solution.

```

Comment:
Suggestions:

- Since you're using "Azure ML" in section titles (instead of the full name), let's define it
- Use sentence case (https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#when-in-doubt-dont-capitalize and https://docs.microsoft.com/en-us/style-guide/capitalization)
- Start with talking about Azure ML components (as a suite of products) and then describe what a component is
- "Lifecycle" (one word - https://docs.microsoft.com/en-us/search/?scope=Style%20guide&terms=lifecycle)
- Define ML before using "ML"

```suggestion
Azure Machine Learning (Azure ML) components are pipeline components that integrate with [Azure ML](https://docs.microsoft.com/en-us/azure/machine-learning/) to manage the lifecycle of your machine learning (ML) models to improve the quality and consistency of your machine learning solution. A pipeline component is a self-contained set of code that performs one step in the ML workflow. A component is analogous to a function, in that it has a name, parameters, return value and a body. 
```